spec/integration/hanami/router/namespace_spec.rb

```diff
@@ -411,5 +411,19 @@
         expect(@app.request('DELETE', '/settings/avatar', lint: true).body).to eq('Avatar::Destroy')
       end
     end
+
+    describe 'mount' do
+      before do
+        @router.namespace 'api' do
+          mount Backend::App, at: '/backend'
+        end
+      end
+
+      [ 'get', 'post', 'delete', 'put', 'patch', 'trace', 'options', 'link', 'unklink' ].each do |verb|
```

Comment:
just style question, maybe `%w[]`?

Reply:
We don't enforce style yet on this code base.